.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: 🐛 Report a bug
+description: '___'
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for reporting a bug! ⛰
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: "Please confirm the following:"
+      options:
+        - label: I am sure the bug is caused by Refined GitHub, I already tried disabling the extension first. (optionally, I also disabled any related userscripts/userstyles and/or in a fresh browser profile if needed).
+          required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the problem and how to replicate it
+      value:
```

Comment:
Any field that is just the default should be dropped. Does it work without it?
```suggestion
```

Reply:
Yes, it works.